float32 distance
---
casper_auto_msgs/MapBasepath[] all_basepaths